@ecomplus/storefront-components/src/html/TheProduct.html

```diff
@@ -167,10 +167,7 @@
               v-if="isKit"
               class="product__kit"
             >
-              <slot
-                name="kit"
-                v-bind="{ kitItems }"
-              >
+              <slot name="kit" v-bind="{ kitItems }">
```

Comment:
```suggestion
              <slot
                name="kit"
                v-bind="{ kitItems }"
              >
```

https://v2.vuejs.org/v2/style-guide/?redirect=true#Multi-attribute-elements-strongly-recommended 💅🏾

Reply:
ok